### PR TITLE
fix(cloudwatchslack): change topic priority

### PR DESCRIPTION
**Pull request by roleyfoley**
## Intent of Change

- Bug fix (non-breaking change which fixes an issue)

## Description

- Sets the deployment priority to ensure the topic is deployed before other solution components which are creating alerts that need the topic
- Updates the component format to align with the latest versions

## Motivation and Context

Reduces the need to redeploy components to enable the alerts as the topic is required by cloudwatch to set the alert action

## How Has This Been Tested?

tested locally

### Prerequisite PRs:

- None

### Dependent PRs:

- None

### Consumer Actions:

- None

### CODE REVIEW

File: aws/modules/cloudwatchslack/module.ftl [hunk 1]
```diff
@@ -102,48 +102,41 @@
...
+                                            "Tier" : tier,
+                                            "Component" : lambdaName,
+                                            "Function" : "send"
                                         }
                                     }
                                 }
                             }
                         },
                         lambdaName : {
                             "lambda" : {
```
Comment (ml019) [inline aws/modules/cloudwatchslack/module.ftl:121]: Given the explicit use of the `attribute`, I think this level in the config can be removed?

**Review by ml019 — CHANGES_REQUESTED** [review]

File: aws/modules/cloudwatchslack/module.ftl (hunk 1 above)
Comment (roleyfoley) [reply aws/modules/cloudwatchslack/module.ftl:121]: Where do you mean attribute? couldn't see it mentioned

Comment (ml019) [reply aws/modules/cloudwatchslack/module.ftl:121]: Line 121 is the type based attribute "lambda", line 122 is the type attribute. I noticed elsewhere you'd dropped the type attribute (e.g. "topic") so for consistency we should drop this one.

Comment (roleyfoley) [reply aws/modules/cloudwatchslack/module.ftl:121]: Ah right! sorry missed that will fix them up

**Review by ml019 — APPROVED** [review]

**Pull request merged**